factory/reader.js

```diff
@@ -35,7 +70,7 @@ function showContent(url) {
   requestContent(url).then(function (body) {
     var content = parseContent(body);
     printArticle(content);
-  }, function () {
+  }, function (e) {
```

Comment:
Hi,

Please remove it.

I had to pass it in and log the specific error when I was debugging.
I was using the :eq selector which unfortunately cheerio doesn't support and was throwing errors, so I worked around it.
You can see the .first().text() on line 29, I wouldn't have needed it if I could put the :eq(0) on the H4 selector.

I removed the log code but I forgot to remove the e variable.